### PR TITLE
docs: add zh proxy-control.md&modify other doc error

### PR DESCRIPTION
**Pull request by liangliang4ward**
### What this PR does / why we need it:

add zh proxy-control.md & modify other doc error

### Pre-submission checklist:

* [ ] Did you explain what problem does this PR solve? Or what new features have been added?
* [ ] Have you added corresponding test cases?
* [ ] Have you modified the corresponding document?
* [ ] Is this PR backward compatible? **If it is not backward compatible, please discuss on the [mailing list](https://github.com/apache/apisix/tree/master#community) first**

### CODE REVIEW

**Review by tzssangglass — APPROVED** [review]

File: docs/zh/latest/plugins/proxy-control.md [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+- [**属性**](#属性)
+- [**如何启用**](#如何启用)
+- [**测试插件**](#测试插件)
+- [**禁用插件**](#禁用插件)
+
+## 名称
+
+`proxy-control` 能够动态地控制Nginx代理的行为。
+
+**这个插件需要APISIX在 [APISIX-OpenResty](../how-to-build.md#step-6-build-openresty-for-apache-apisix)上运行。**
```
Comment (spacewander) [inline docs/zh/latest/plugins/proxy-control.md:36]:
```suggestion
**这个插件需要APISIX在 [APISIX-OpenResty](../how-to-build.md#步骤6：为-Apache-APISIX-构建-OpenResty)上运行。**
```

Comment (liangliang4ward) [reply docs/zh/latest/plugins/proxy-control.md]: done;

Comment (liangliang4ward) [reply docs/zh/latest/plugins/proxy-control.md]: change to `步骤6为-Apache-APISIX-构建-OpenResty` be better.

Comment (spacewander) [reply docs/zh/latest/plugins/proxy-control.md]: The thing is a bit tricky there. The anchor in the website is different from the one in GitHub.

Here is the link in website:
https://apisix.apache.org/zh/docs/apisix/how-to-build/#%E6%AD%A5%E9%AA%A46%EF%BC%9A%E4%B8%BA-apache-apisix-%E6%9E%84%E5%BB%BA-openresty

We prefer to use the anchor style in the website.

**Review by spacewander — APPROVED** [review]

File: docs/zh/latest/plugins/proxy-control.md [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+
+`proxy-control` 能够动态地控制Nginx代理的行为。
+
+**这个插件需要APISIX在 [APISIX-OpenResty](../how-to-build.md#步骤6：为-Apache-APISIX-构建-OpenResty)上运行。**
+
+## 属性
+
+| 名称      | 类型          | 必选项 | 默认值    | 有效值                                                                    | 描述                                                                                                                                         |
+| --------- | ------------- | ----------- | ---------- | ------------------------------------------------------------------------ | --------------------------------------------------------------------------------------------------------------------------------------------------- |
+| request_buffering | boolean        | 可选    |  true            |  | 动态设置 `proxy_request_buffering` |
```
[the remark this answers is not in the document]
Comment (liangliang4ward) [reply docs/zh/latest/plugins/proxy-control.md]: done

**Review by tokers — APPROVED** [review]

File: docs/zh/latest/plugins/traffic-split.md [hunk 3]
```diff
@@ -224,7 +224,7 @@ world 1981
 
 ### 蓝绿发布
 
-通过请求头获取 `match` 规则参数(也可以通过请求参数获取或NGINX变量)，在 `match` 规则匹配通过后，表示所有请求都命中到插件配置的 upstream ，否则所以请求只命中 `route` 上配置的 upstream 。
+通过请求头获取 `match` 规则参数(也可以通过请求参数获取或NGINX变量)，在 `match` 规则匹配通过后，表示所有请求都命中到插件配置的 upstream ，否则所有请求只命中 `route` 上配置的 upstream 。
```
Comment (zaunist) [inline docs/zh/latest/plugins/traffic-split.md:227]:
```suggestion
通过请求头获取 `match` 规则参数(也可以通过请求参数获取或 NGINX 变量)，在 `match` 规则匹配通过后，表示所有请求都命中到插件配置的 upstream ，否则所有请求只命中 `route` 上配置的 upstream 。
```

File: docs/zh/latest/plugins/proxy-control.md [hunk 4]
```diff
@@ -0,0 +1,93 @@
...
+
+- [**名称**](#名称)
+- [**属性**](#属性)
+- [**如何启用**](#如何启用)
+- [**测试插件**](#测试插件)
+- [**禁用插件**](#禁用插件)
+
+## 名称
+
+`proxy-control` 能够动态地控制Nginx代理的行为。
```
Comment (tzssangglass) [inline docs/zh/latest/plugins/proxy-control.md:34]:
```suggestion
`proxy-control` 能够动态地控制 Nginx 代理的行为。
```

File: docs/zh/latest/plugins/traffic-split.md (hunk 3 above)
Comment (liangliang4ward) [reply docs/zh/latest/plugins/traffic-split.md]: done

**Review by tzssangglass — APPROVED** [review]

**Review by spacewander — APPROVED** [review]

**Comment by spacewander** [comment]
```
rg 为-Apache-APISIX-构建-OpenResty
docs/zh/latest/admin-api.md
588:以下特性需要 APISIX 运行于 [APISIX-OpenResty](./how-to-build.md#步骤6：为-Apache-APISIX-构建-OpenResty)：
597:这个特性需要 APISIX 运行于 [APISIX-OpenResty](./how-to-build.md#步骤6：为-Apache-APISIX-构建-OpenResty)。

docs/zh/latest/mtls.md
69:你需要构建 [APISIX-OpenResty](./how-to-build.md#步骤6：为-Apache-APISIX-构建-OpenResty)，并且需要在配置文件中设定 `etcd.tls` 来使 ETCD 的双向认证功能正常工作。
157:该功能需要 APISIX 运行在 [APISIX-OpenResty](./how-to-build.md#步骤6：为-Apache-APISIX-构建-OpenResty) 上。

docs/zh/latest/plugins/proxy-control.md
36:**这个插件需要APISIX在 [APISIX-OpenResty](../how-to-build.md#步骤6：为-Apache-APISIX-构建-OpenResty)上运行。**

docs/zh/latest/plugins/gzip.md
36:**该插件要求 `APISIX` 运行在 [APISIX-OpenResty](./how-to-build.md#步骤6：为-Apache-APISIX-构建-OpenResty) 上。**

docs/zh/latest/plugins/dubbo-proxy.md
40:如果你正在使用 `OpenResty`, 你需要编译它来支持 `dubbo`, 参考 [如何编译](../how-to-build.md#步骤6：为-Apache-APISIX-构建-OpenResty)。

docs/zh/latest/plugins/real-ip.md
38:**该插件要求 `APISIX` 运行在 [APISIX-OpenResty](./how-to-build.md#步骤6：为-Apache-APISIX-构建-OpenResty) 上。**
```

Would you like to fix the link in gzip and real-ip?

**Comment by liangliang4ward** [comment]
> Would you like to fix the link in gzip and real-ip?

With pleasure.

**Comment by liangliang4ward** [comment]
> ```
> rg 为-Apache-APISIX-构建-OpenResty
> docs/zh/latest/admin-api.md
> 588:以下特性需要 APISIX 运行于 [APISIX-OpenResty](./how-to-build.md#步骤6：为-Apache-APISIX-构建-OpenResty)：
> 597:这个特性需要 APISIX 运行于 [APISIX-OpenResty](./how-to-build.md#步骤6：为-Apache-APISIX-构建-OpenResty)。
> 
> docs/zh/latest/mtls.md
> 69:你需要构建 [APISIX-OpenResty](./how-to-build.md#步骤6：为-Apache-APISIX-构建-OpenResty)，并且需要在配置文件中设定 `etcd.tls` 来使 ETCD 的双向认证功能正常工作。
> 157:该功能需要 APISIX 运行在 [APISIX-OpenResty](./how-to-build.md#步骤6：为-Apache-APISIX-构建-OpenResty) 上。
> 
> docs/zh/latest/plugins/proxy-control.md
> 36:**这个插件需要APISIX在 [APISIX-OpenResty](../how-to-build.md#步骤6：为-Apache-APISIX-构建-OpenResty)上运行。**
> 
> docs/zh/latest/plugins/gzip.md
> 36:**该插件要求 `APISIX` 运行在 [APISIX-OpenResty](./how-to-build.md#步骤6：为-Apache-APISIX-构建-OpenResty) 上。**
> 
> docs/zh/latest/plugins/dubbo-proxy.md
> 40:如果你正在使用 `OpenResty`, 你需要编译它来支持 `dubbo`, 参考 [如何编译](../how-to-build.md#步骤6：为-Apache-APISIX-构建-OpenResty)。
> 
> docs/zh/latest/plugins/real-ip.md
> 38:**该插件要求 `APISIX` 运行在 [APISIX-OpenResty](./how-to-build.md#步骤6：为-Apache-APISIX-构建-OpenResty) 上。**
> ```
> 
> Would you like to fix the link in gzip and real-ip?

https://github.com/apache/apisix/pull/6417

**Pull request merged**